ui/app/clinical/displaycontrols/treatmentData/views/treatmentTable.html

```diff
@@ -7,7 +7,7 @@ <h2 ng-if="::params.translationKey" class="section-title">{{::params | titleTran
             <thead>
             <th colspan="4" ng-if="::!isOtherActiveSection(drugOrderSection.visitDate)">
                 {{::'VISIT_ON_TRANSLATION_KEY'|translate}} {{::drugOrderSection.visitDate | bahmniDate}}
-                <div class="share-prescription-icon" ng-if="::!params.visitUuids" bm-pop-over autoclose="true">
+                <div class="share-prescription-icon" ng-if="::!params.visitUuids" bm-pop-over autoclose="true" title="Share or print the Prescription">
```

Comment:
Can we remove the hardcoded tooltip and have it as translation key? Also the casing is not proper.